app/MMB/package.json

```diff
@@ -11,6 +11,8 @@
     "@kiwicom/mobile-pdf": "^0",
     "@kiwicom/mobile-relay": "^0",
     "@kiwicom/mobile-shared": "^0",
+    "@kiwicom/rnmodules": "^0",
+    "@kiwicom/react-native-app-hotels":"^0",
```

Comment:
Why was that dependency added to MMB?

Reply:
I bet because you are using hotels somewhere inside MMB?

Reply:
> Hmmm, not sure...

https://github.com/kiwicom/mobile/blob/34684baf3557f0fef451c0f555505bc948855351/app/MMB/src/navigation/NavigationStack.js#L11 :)